lib/class-wp-theme-json-gutenberg.php

```diff
@@ -785,6 +785,9 @@ protected static function sanitize( $input, $valid_block_names, $valid_element_n
 	 * @return string The new selector.
 	 */
 	protected static function append_to_selector( $selector, $to_append, $position = 'right' ) {
+		if ( ! str_contains( ',', $selector ) ) {
```

Comment:
Perhaps the order of the arguments is reversed?
See: https://www.php.net/manual/en/function.str-contains.php